src/common/rule-based-view-model-provider.ts

```diff
@@ -1,15 +1,28 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { AllRuleResultStatuses, CardRuleResult, CardRuleResultsByStatus, CardRuleResultStatus } from './types/store-data/card-view-model';
+import { CardSelectionViewData } from 'common/get-card-selection-view-data';
+import { includes } from 'lodash';
+import {
+    AllRuleResultStatuses,
+    CardResult,
+    CardRuleResult,
+    CardRuleResultsByStatus,
+    CardRuleResultStatus,
+} from './types/store-data/card-view-model';
 import { UnifiedResult, UnifiedRule } from './types/store-data/unified-data-interface';
 
-export type GetUnifiedRuleResultsDelegate = (rules: UnifiedRule[], results: UnifiedResult[]) => CardRuleResultsByStatus;
+export type GetUnifiedRuleResultsDelegate = (
```

Comment:
This sounds like it's transforming some other input type into unified rule results, but it seems to actually be transforming unified rules/results into `CardRuleResults`. Maybe this (and the similar nearby functions) would be better named `GetCardRuleResultsDelegate`?